README.md

```diff
@@ -1 +1,79 @@
-# final-capstone-back-end
\ No newline at end of file
+# Slowppi Backend
+
+Ruby on Rails API for reservations.
+
+## Table of contents
+
+- [Overview](#overview)
+  - [API Endpoint](#api-endpoint)
+  - [Frontend](#frontend)
+  - [Available scripts](#available-scripts)
+- [My process](#my-process)
+  - [Built with](#built-with)
+- [Author](#author)
+
+## Overview
+
+### API Endpoint
+
+- API Endpoint: `slowppi-services.herokuapp.com/v1/`
+
+### Frontend
+
+- [Repository](https://github.com/alonsofl/slowppi-frontend)
+
+### Available scripts
+
+- Clone this repository
+```bash
+$ git clone git@github.com:leonardodiasb/final-capstone-back-end.git
+$ cd final-capstone-back-end/
+```
+
+- Seed the database
+```bash
+$ rails db:seed
+```
+
+- Start Rails server
+```bash
+$ rails s
+```
+
+- Start Rails console
+```bash
+$ rails c
+```
+
```

Comment:
You did a great job with these instructions. We have some instructions that could be a great addition to assist anyone interacting with your code with a seamless setup. Please consider the suggestions below. Thank you! 
```suggestion

- Create the database
```bash
$ rails db:create
```

- Make the migrations
```bash
$ rails db:migrate
```

- Seed the database
```bash
$ rails db:seed
```

- Start Rails server
```bash
$ rails s
```

- Start Rails console
```bash
$ rails c
```

- Run the `Intergration` & `Unit` tests

```bash
$ rspec
```

```